sim/uhdm-integration-verilator/condarc

```diff
@@ -0,0 +1,2 @@
+channels:
+  - conda-forge
```

Comment:
Dump the whole `condarc` file. `conda-forge` won't be required to install `surelog-uhdm` and `verilator-uhdm` after changing their `condarc` files.

Reply:
Dumped